pkg/cloudprovider/providers/azure/azure_storageaccount.go

```diff
@@ -113,6 +113,8 @@ func (az *Cloud) ensureStorageAccount(accountName, accountType, location, genAcc
 				accountName, az.ResourceGroup, location, accountType)
 			cp := storage.AccountCreateParameters{
 				Sku: &storage.Sku{Name: storage.SkuName(accountType)},
+				// set as StorageV2 according to https://docs.microsoft.com/en-us/rest/api/storagerp/StorageAccounts/Update
```

Comment:
nit: Also add a comment: "Currently only StorageV2 supported"

Reply:
hmm, this is only for create, so all 3 kinds are supported here. So it's better changing the link to https://docs.microsoft.com/en-us/azure/storage/common/storage-account-options and add a comment of why StorageV2 is chosen

Reply:
This is exactly what i meant.
The comment should be 'StorageV2' is chosen by purpose, not limited by api.

Reply:
fixed by new comment